src/views/home-view.js

```diff
@@ -123,16 +123,6 @@ class HomeView extends ViewMixin(ReduxMixin(PolymerElement)) {
      src="../../images/index/hero/hero_image_web_components@2600.png" alt="Banner Image">
 
 <div class="container">
-    <div class="row">
-        <div class="row-item">
-            <h1><b>Up to 98% faster integration</b></h1>
-            <p style="text-align: center">
-                With the new FACT-Finder Web Components
-            </p>
-        </div>
-    </div>
-
-    <hr>
```

Comment:
Removing the `<hr>` makes the tiles from below touch the main image. This looks a little awkward, don't you agree?
Keeping the `<hr>` might look a little odd, too, but placing a margin of `30px` (double of `.container`'s left-padding) between the image and `.container` looks quite pleasing.
Didn't check if `.container` is used somewhere else but it might be best to put the margin on the image's bottom anyway.
What's the opinion of the designer in you?